docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -857,6 +857,81 @@
+      which endpoints and ports components are permitted to connect to.</para>
+
+      <para>Most HPCC Systems components have their own auto-generated network
+      policies. The generated network policies typically work to limit ingress
+      and egress to inter-component communication, or expected external
+      service ingress only.</para>
+
+      <para>For instance, in a default deployed system with network policies
+      enforced, a query running (on hThor, Thor, or Roxie), will not be able
+      to connect with a 3rd party service, such as an LDAP service or log
+      stack. </para>
+
+      <para>Deploying generates a "default-deny-all-but-dns" which prevents
+      all but DNS, however, we also generate "allow-apiserver" (also with no
+      pod-selector).</para>
+
+      <para>In the default configuration, any pod with Kube API access will
+      also have access to any external endpoint. This is because a
+      <emphasis>NetworkPolicy</emphasis> is generated for egress access for
+      components that need access to the Kube API. However,
+      <emphasis>global.egress.kubeApiCidr</emphasis> and
+      <emphasis>global.egress.kubeApiPort</emphasis> in the values.yaml should
+      be configured in a secure system to lock this egress access down so that
+      it only exposes egress access to the Kube API endpoint.</para>
+
+      <para>We have added a mechanism similar to the visibilities definitions,
+      which allows named egress sections, which can then be referenced per
+      component.</para>
+
+      <para>For example:</para>
+
+      <programlisting>global:
+  egress:
+    engineEgress:
+    - to:
+        - ipBlock:
+            cidr: 201.13.21.0/24
+        - ipBlock:
+            cidr: 142.250.187.0/24
+      ports:
+        - protocol: TCP
+          port: 6789
+        - protocol: TCP
+          port: 7890
+...
+
+thor:
+...
+  egress: engineEgress</programlisting>
+
+      <para>Note that the name 'engineEgress' is an arbitrary name, any name
+      can be chosen, and any number of these named egress sections can be
+      defined.</para>
+
+      <para>For more information, please see the <emphasis>egress:</emphasis>
+      section in the default stock/delivered HPCC Systems YAML file. The
+      <emphasis>values.yaml</emphasis> file can be found under the helm/hpcc/
+      directory on the HPCC Systems github repository:</para>
+
+      <para><ulink
+      url="https://github.com/hpcc-systems/HPCC-Platform">https://github.com/hpcc-systems/HPCC-Platform</ulink></para>
+
+      <para>If global.egress.<emphasis>kubeApiCidr</emphasis> and
+      global.egress<emphasis>.kubeApiPort</emphasis> values were set, then it
+      would egress enforce access to only the Kubectl API cidr as intended.
+      However without those values set, it allows any pod (without another
+      more restrictive network policy) to egress to any endpoint.</para>
```

Comment:
I also think this paragraph should be removed, it is basically repeating what is now said on lines 882-889,
and doesn't flow from previous paragraph.